airbyte-integrations/connector-templates/source-python-http-api/source_{{snakeCase name}}/source.py.hbs

```diff
@@ -30,68 +30,112 @@ import requests
+
+        For example, if the API accepts a 'page' parameter to determine which page of the result to return, and a response from the API contains a
+        'page' number, then this method should probably return a dict {'page': response.json()['page'] + 1} to increment the page count by 1.
+        The request_params method should then read the input next_page_token and set the 'page' param to next_page_token['page'].
+
+        :param response: the most recent response from the API
+        :return If there is another page in the result, a mapping (e.g: dict) containing information needed to query the next page in the response.
+                If there are no more pages in the result, return None.
         """
         return None
 
     def request_params(self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None) -> MutableMapping[str, Any]:
         """
-        TODO: Override this method to define any query parameters to be set. Remove this method if not needed. Usually contains common params e.g. pagination size etc.
+        TODO: Override this method to define any query parameters to be set. Remove this method if you don't need to define request params.
+        Usually contains common params e.g. pagination size etc.
         """
         return {}
 
 
-class MultipleShotPotatoGun({{properCase name}}Stream):
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+    """
+    TODO: Override this method to define how a response is parsed.
+    :return an iterable containing each record in the response
+    """
+        yield {}
+
+
+class Customers({{properCase name}}Stream):
     """
-    TODO: Change class name to match the table/data source this strema corresponds to.
+    TODO: Change class name to match the table/data source this stream corresponds to.
     """
 
     def path(self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None) -> str:
         """
-        TODO: Override this method to define the path this stream corresponds to. E.g. if the url is https://potatogun.com/v1/multiple then this should
-        return "multiple". Required.
-        """
-        return 'fill_in_this_stream_specific_path'
-
-    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
-        """
-        TODO: Override this method to define how a response is translated into one, or more, AirbyteRecordMessages.
-        Returns an iterable of the response's field, by default. Remove this method if not needed.
+        TODO: Override this method to define the path this stream corresponds to. E.g. if the url is https://example-api.com/v1/customers then this
+        should return "customers". Required.
         """
-        yield {}
+        return 'customers'
 
-
-# Incremental Stream
```

Comment:
since we have `basic full refresh stream` above, should this be `basic incremental stream`?

Reply:
I added it.